### ISSUE TITLE
Fern buffer is broken when I invoke `<Plug>(fern-aciton-open:vsplit)`

### ISSUE THREAD

**Issue by asmka** [lambdalisue/fern.vim#413]
## Description
Fern buffer is broken when I invoke `<Plug>(fern-aciton-open:vsplit)` under specific confitions.

![fern-issue-01](https://user-images.githubusercontent.com/33749863/161778757-9337291b-1fbb-4905-b63f-908b450ea40a.png)
![fern-issue-02](https://user-images.githubusercontent.com/33749863/161778768-24a8a0a9-7fec-4b79-9309-1fe9319ad282.png)

## Environment
- Editor: Neovim 0.6.1
- OS: Ubuntu 20.04

## Fern setting
```Lua
vim.g['fern#disable_default_mappings'] = 1
vim.g['fern#disable_drawer_smart_quit'] = 1

function _G.FernInit()
  vim.cmd[[nmap <buffer> <CR> <Plug>(fern-action-open:edit)]]
  vim.cmd[[nmap <buffer> s <Plug>(fern-action-open:vsplit)]]
end

vim.cmd[[augroup FernEvents]]
vim.cmd[[  autocmd!]]
vim.cmd[[  autocmd FileType fern call v:lua.FernInit()]]
vim.cmd[[augroup END]]
```

## Procedure
1. Boot Neovim
2. `:Fern . -drawer -stay`
3. `:q`
4. Open file with `<CR>` key (`<Plug>(fern-action-open:edit)`)
5. `:Fern . -drawer`
6. Open file with `s` key (`<Plug>(fern-action-open:vsplit)`)

**Comment by lambdalisue** [comment]
Reproduced with the procedure and minimal vimrc 

```vim
if exists('+compatible') && &compatible
  set nocompatible
endif

" Disable Vim's native pack feature
set packpath=

" Clone https://github.com/lambdalisue/fern.vim in somewhere
" and specify that directory to the below
set runtimepath^=~/ghq/github.com/lambdalisue/fern.vim

filetype plugin indent on
syntax on
"----------------------------------------------------------------

nnoremap ; :
nnoremap : ;

let g:fern#disable_default_mappings = 1
let g:fern#disable_drawer_smart_quit = 1

function! s:my_fern() abort
  nmap <buffer> <CR> <Plug>(fern-action-open:edit)
  nmap <buffer> s <Plug>(fern-action-open:vsplit)
endfunction

augroup my_fern
  autocmd!
  autocmd FileType fern call s:my_fern()
augroup END

"----------------------------------------------------------------
echomsg "Custom minimal vimrc has loaded"
```

**Comment by asmka** [comment]
@lambdalisue 
I confirmed the problem was resolved by #414.
Thank you for your quick handling! 🙇‍♀️
